tools/keycloak/sync/realm.local.json

```diff
@@ -17,7 +17,10 @@
         "Protocol": "openid-connect",
         "BearerOnly": true,
         "AuthorizationServicesEnabled": false,
-        "RedirectUris": ["http://localhost:5000/api/*"],
+        "RedirectUris": [
+          "http://localhost:5000/api/*",
+          "http://localhost:61081/api/*"
```

Comment:
These changes make it easier for my local environment to work with the Keycloak Sync Tool.